requirements.txt

```diff
@@ -99,10 +101,15 @@ Sphinx~=5.0.2
 sphinx-rtd-theme~=1.0.0
 sphinxcontrib-applehelp~=1.0.2
 sphinxcontrib-devhelp~=1.0.2
+sphinx-favicon~=0.2
```

Comment:
does broadbean have a favicon?

Reply:
not to my knowledge I removed this dep and we can pull in back in if the docs require a facicon in a later PR